auth/ReceiverKeyStateMachine.m

```diff
@@ -179,8 +183,7 @@ function disp(obj)
         end
 
         function purge_expired_keys(obj, current_time, level)
-            % removes all expired keys of input level (or all levels if no
-            % input level provided)
+            % Removes all expired keys of input level (or all levels if no input level provided).
```

Comment:
The use of nargin to filter if you are going to purge all key levels or just one that is specified is interesting. Is this conventional, I could see myself forgetting a parameter on accident and in that case it will delete keys we didn't intend.

Reply:
well its not that bad if expired keys are removed...I did this to save lines of code...I don't need another function to purge all 3 and now I don't have to call essentially the same method 3 times elsewhere in the code